### PR TITLE
PubChem RDF tools

### PR DESCRIPTION
**Pull request by xtracko**
This PR enables downloading [PubChem RDF](https://pubchemdocs.ncbi.nlm.nih.gov/rdf) and building our compound database which can be later used in annotation.

### CODE REVIEW

**Comment by martenson** [comment]
@xtracko Is there a possibility for adding a basic test?

**Comment by xtracko** [comment]
I am working on adding the test right now. I will commit it shortly after.

**Comment by martenson** [comment]
@xtracko could you please commit the test so we can move this forward? Thank you.

File: tools/pubchem/pubchem_build.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <requirement type="package" version="1.2.1">pandas</requirement>
+        <requirement type="package" version="4.0.30">pyodbc</requirement>
+        <requirement type="package" version="0.5.4">python-snappy</requirement>
+        <requirement type="package" version="3.8">python</requirement>
+    </requirements>
+
+    <command detect_errors="aggressive"><![CDATA[
+        tar -xvf ${pubchem_rdf}
+
+        python '${__tool_directory__}/pubchem_build.py'
```
Comment (martenson) [inline tools/pubchem/pubchem_build.xml:16]: Galaxy treats this as one command, hence the error `tar: unrecognized option '--workers' Try 'tar --help' or 'tar --usage' for more information.` when trying to run this. We could chain it with some logical operators like `&&`, extract to an external script, etc.

**Comment by martenson** [comment]
The tool now runs without error. However it is still unclear to me with what data to test it, 4 random RDF sets I've chosen produce 0 bytes parquet dataset.

**Comment by hechth** [comment]
What is the current status of this? Can this be merged, does it need further work? If so, what are the issues?

So that maybe we can move this a bit further :)

**Comment by martenson** [comment]
@hechth fwiw it cannot be merged in this state since there are unanswered questions from my review.

**Comment by hechth** [comment]
@martenson I think we can close this for now - in case it would be necessary, we can re-open.

**Comment by martenson** [comment]
@hechth agreed, closed

**Pull request closed without merging**